CONTRIBUTING.md

```diff
@@ -59,6 +59,17 @@ of Windows.
 * Visual Studio 2017+ or Visual Studio Code
 * .NET Framework 4.6+
 
+### Public Api
+
+It is critical to keep public API surface small and clean. This repository is
+using `Microsoft.CodeAnalysis.PublicApiAnalyzers` to validate the public APIs.
+This analyzer will check if you changed a public property/method so the change
+will be easily spotted in pull request. It will also ensure that OpenTelemetry
+doesn't expose APIs outside of the library primary concerns like a generic
+helper methods.
+
+Use intellisence to update the publicApi.
```

Comment:
It is unclear that this line is about.

Some nits:
* `IntelliSense`, according to https://docs.microsoft.com/visualstudio/ide/using-intellisense.
* `publicApi` should be a file or something that can be edited/updated.

Reply:
Hi @reyang , just pushed a new commit adding how to enable / configure. Let me know what do you think